packages/@webex/providers-cc-widget/webpack.config.js

```diff
@@ -0,0 +1,42 @@
+const path = require('path');
+const glob = require('glob');
+
+module.exports = {
+  mode: 'development',
+  entry: glob.sync('./src/**/*.ts'),
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: '[name].js',
+    libraryTarget: 'commonjs2',
+  },
+  resolve: {
+    extensions: ['.ts', '.tsx', '.js'],
+  },
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: 'ts-loader',
+      },
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[path][name].js',
+              context: 'src',
+            },
+          },
+        ],
+        enforce: 'post',
+      },
+    ],
+  },
+  externals: {
+    react: 'react',
+    'react-dom': 'react-dom',
+  },
+};
```

Comment:
Can we create a base webpack config file, import and use it inside all these packages?

Reply:
Created a base config